assets/js/icons/library/arrow-back.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { SVG } from 'wordpress-components';
+import { SVG } from '@wordpress/components';
```

Comment:
If the icons are used on the frontend, doesn't this mean _all_ of `@wordpress/components` will get loaded which means an increase in overall weight on the assets loaded (this comment applies to other files where this change was made too).

Reply:
Possibly, but this usage in the icon library did cause a CD issue. I will look it up tomorrow.

Reply:
Fixed by removing module replacements https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4025/commits/a9471a5b8234a67593c8813078d1bd6982b574c3